pkg/backends/otlp/internal/data/event_response.go

```diff
@@ -0,0 +1,34 @@
+package data
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	v1export "go.opentelemetry.io/proto/otlp/collector/logs/v1"
+	"go.uber.org/multierr"
+	"google.golang.org/protobuf/proto"
+)
+
+func ProcessEventsResponse(resp *http.Response) error {
+	buf, errs := io.ReadAll(resp.Body)
+	if errs != nil {
+		return errs
+	}
+
+	if resp.StatusCode/100 != 2 {
+		errs = multierr.Append(errs, fmt.Errorf("returned a non 2XX status code of %d", resp.StatusCode))
+	}
+
+	var response v1export.ExportLogsServiceResponse
+	if err := proto.Unmarshal(buf, &response); err != nil {
+		errs = multierr.Append(errs, fmt.Errorf("failed to unmarshal response: %w", err))
+		return errs
+	}
+
+	if ps := response.PartialSuccess; ps != nil && ps.ErrorMessage != "" {
+		errs = multierr.Append(errs, fmt.Errorf("failed to send events: %s", ps.ErrorMessage))
```

Comment:
Do we also want to record here the # of rejected log records similar to datapoints?

Reply:
I removed it.. 'cause unlike data points, each request only send one logRecord. So there can't be more than 1 dropped events.